packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -262,7 +262,13 @@ const AccountSwitcher = props => {
     };
 
     const canOpenMulti = () => {
-        if (props.is_eu) return false;
+        if (props.is_eu) {
+            if (props.landing_company_shortcode === 'malta') return false;
```

Comment:
do we have to show `Manage accounts` button only for `MF` and not for `MLT`?

Reply:
We want to show `Manage account` to all real accounts (where applicable).
I've updated this part.